analysis/wwz/wwz4l.py

```diff
@@ -476,28 +479,32 @@ def process(self, events):
             cleanedJets = os_ec.get_cleaned_collection(l_wwz_t,jets)
             jetptname = "pt_nom" if hasattr(cleanedJets, "pt_nom") else "pt"
 
-            # Jet Veto Maps
```

Comment:
Can you explain why this veto map stuff is removed? Is it moved to somewhere else, or done in a different way?

Reply:
Oh never mind, seems to be moved.